avm/ptn/dev-ops/cicd-agents-and-runners/tests/e2e/private-github-existing-vnet-aca/dependencies.bicep

```diff
@@ -0,0 +1,143 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Optional. The naming prefix.')
+param namePrefix string = 'dev'
```

Comment:
Same comment as for the `aci` counterpart

Reply:
Resolved